CMakeLists_files.cmake

```diff
@@ -14,6 +14,7 @@ list( APPEND MAIN_SOURCE_FILES
 
 list (APPEND PUBLIC_HEADER_FILES
         opm/output/OutputWriter.hpp
+        opm/output/Wells.hpp
```

Comment:
Two new files, just one change in the file list. You must add the test program, too.